app/src/main/java/ar/scacchipa/twittercloneapp/data/repository/ICredentialRepository.kt

```diff
@@ -1,9 +1,12 @@
 package ar.scacchipa.twittercloneapp.data.repository
 
+import ar.scacchipa.twittercloneapp.domain.model.Credential
 import ar.scacchipa.twittercloneapp.domain.model.ResponseDomain
 
-interface IAuthorizationRepository {
-    suspend fun requestAccessToken(
+interface ICredentialRepository {
+    suspend fun storeLocalCredential(credential: Credential): Boolean
+    fun recoverLocalCredential(): Credential?
```

Comment:
same as before, I probably will use a suspend fun for this

Reply:
All Repository layer function was converted to suspend function